daemon/start.go

```diff
@@ -158,6 +158,12 @@ func (daemon *Daemon) containerStart(container *container.Container) (err error)
 			container.SetExitCode(126)
 		}
 
+		// attempted to mount a file onto a directory, or a directory onto a file, maybe from user specified bind mounts
+		if strings.Contains(errDesc, syscall.ENOTDIR.Error()) {
+			errDesc += ": check that all specified host paths exist and are the correct file type for all volumes"
```

Comment:
Yes, I saw the comment above doing roughly that. I think we can use this then - after all, it's just an error message :smile: (we can always improve in future)